docs/tasks/administer-cluster/kubeadm-upgrade-1-8.md

```diff
@@ -42,6 +42,13 @@ $ curl -sSL https://dl.k8s.io/release/${VERSION}/bin/linux/${ARCH}/kubeadm > /us
 $ chmod a+rx /usr/bin/kubeadm
 ```
 
+NOTE: You will notice that `kubeadm` is also shipped in the Kubernetes
```

Comment:
Note looks good, but could you toss in the relevant format structure for it so it stands out appropriately? (from https://kubernetes.io/docs/home/contribute/style-guide/ - )

format I think you want is:

```
**Note:** The prefix you use is the same text you use in the tag. 
{: .note} <!-- This tag must appear on a new line. -->
```

Reply:
+1 (possibly worth being called out with the "caution" tag rather than the "note" tag)